backend/src/install/mod.rs

```diff
@@ -1436,38 +1390,6 @@ pub async fn install_s9pk<R: AsyncRead + AsyncSeek + Unpin + Send + Sync>(
             &receipts.config.update_dependency_receipts,
         )
         .await?;
-    } else if let Some(recovered) = {
```

Comment:
This is why I don't think that just being dumb and removing the recoveredPackages is a good idea